pom.xml

```
@@ -25,7 +25,7 @@
 
     <groupId>io.confluent.kafka.connect</groupId>
     <artifactId>kafka-connect-datagen</artifactId>
-    <version>0.1.5-SNAPSHOT</version>
+    <version>0.1.5</version>
```

@ybyzek we don't want to change this manually, since doing so is an important part of the release process. Yes, that means the Dockerfiles will be broken in the branch until we release, but there's no easy way around that.

```suggestion
    <version>0.1.5-SNAPSHOT</version>
```